spec/operators/windowTime-spec.js

```diff
@@ -108,19 +116,45 @@ describe('Observable.prototype.windowTime', function () {
   it('should emit windows given windowTimeSpan and windowCreationInterval, ' +
   'but outer is unsubscribed early', function () {
     var source = hot('--1--2--^--a--b--c--d--e--f--g--h--|');
-    var unsub =              '              !             ';
+    var subs =               '^          !                ';
     //  100 frames            0---------1---------2------|
     //  50                     ----|
     //  50                               ----|
     //  50                                         ----|
-    var expected =           'x---------y----             ';
+    var expected =           'x---------y-                ';
     var x = cold(            '---a-|                      ');
-    var y = cold(                      '--d--             ');
+    var y = cold(                      '--                ');
+    var unsub =              '           !                ';
     var values = { x: x, y: y };
 
     var result = source.windowTime(50, 100, rxTestScheduler);
 
     expectObservable(result, unsub).toBe(expected, values);
+    expectSubscriptions(source.subscriptions).toBe(subs);
+  });
+
+  it('should dispose window Subjects if the outer is unsubscribed early', function () {
+    var source = hot('--a--b--c--d--e--f--g--h--|');
+    var sourceSubs = '^        !                 ';
+    var expected =   'x---------                 ';
+    var x = cold(    '--a--b--c-                 ');
+    var unsub =      '         !                 ';
+    var values = { x: x };
+
+    var window;
+    var result = source.windowTime(1000, 1000, rxTestScheduler)
+      .do(function (w) { window = w; });
+
+    expectObservable(result, unsub).toBe(expected, values);
+    expectSubscriptions(source.subscriptions).toBe(sourceSubs);
+    rxTestScheduler.schedule(function () {
+      try {
+        window.subscribe();
+      }
+      catch (err) {
+        expect(err.message).toBe('Cannot subscribe to a disposed Subject.');
+      }
+    }, 150);
```

Comment:
Will this test also to use `time` in https://github.com/ReactiveX/RxJS/pull/1032 in later?